### PR TITLE
Address route

### PR DESCRIPTION
**Pull request by Azecko**
`/api/address` takes a `q` parameter that works like the `/api/ldap/ ` route.

### CODE REVIEW

**Comment by Azecko** [comment]
Tests are coming

**Comment by Azecko** [comment]
Tests are all successful.
image

I had to add `sciper`, `firstname`, `name` and `email` in the JSON return so the `score()` function doesn't return errors. (I just added a userMapper)

**Review by williambelle — COMMENTED** [review]
Je propose aussi d'ajouter ton nouveau endpoint dans le README si tu es d'accord ?

File: src/controllers/address.controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const ldapUtil = require('../utils/ldap.util');
+const addressService = require('../services/address.service');
```
[the remark this answers is not in the document]
Comment (Azecko) [reply src/controllers/address.controller.js]: closed by fc28a31

File: src/app.js [hunk 2]
```diff
@@ -39,6 +40,11 @@ if (configApi.enableLdap) {
   app.use('/api/ldap', peopleRouter);
 }
 
+// Address
+if (configApi.enableLdap) {
```
[the remark this answers is not in the document]
Comment (Azecko) [reply src/app.js]: closed by c4dd521

**Review by williambelle — APPROVED** [review]

**Pull request merged**